CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## 0.41.0
 
+* Enhance `KafkaBridge` resource with consumer inactivity timeout and HTTP consumer/producer enablement.
```

Comment:
As this will be part only of the 0.42 release, can you please move this to the 0.42.0 section?